### DISCUSSION TITLE
Sponsor Version rebooting issue

### DISCUSSION THREAD

**Discussion by Celticpure** [alf45tar/PedalinoMini#409] [Q&A]
When I make changes to the Sponsor version via the web interface ie: rename a bank and then try to save it to Pedalino. Pedalino then goes into constant reboot, which results in a complete reinstall. I have tried this numerous times and with the same outcome. I have gone back to the public version for now. 

Any thoughts - maybe i am missing something?

**Comment by alf45tar** [comment]
You need a reset to factory default or erase flash before upload firmware and do not forget to upload system image too.

If you still have reboot attach the log.

**Reply by Celticpure** [reply]
Yes I have done this, I mentioned this in another thread also: https://github.com/alf45tar/PedalinoMini/discussions/405#discussioncomment-4387463

There is no change though, when any change has been made via the Web UI and then saved Pedalino goes into constant reboot searching for WiFi. So I just gave up and flashed the public version for now.

**Reply by alf45tar** [reply]
Can you attach the log?

**Reply by Celticpure** [reply]
Not Sure how to get the log, but from the Monitor window in VSC Studio I am getting the attached loop when I try and name a BANK. I have copied and pasted it into notepad. 

[Pedalino Log 31.12.22.txt](https://github.com/alf45tar/PedalinoMini/files/10327570/Pedalino.Log.31.12.22.txt)
